sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -149,12 +146,59 @@ public HttpPipeline getHttpPipeline() {
      * @param digitalTwinId The Id of the source digital twin.
      * @param relationshipName The name of a relationship to filter to.
      * @return A {@link PagedFlux} of application/json relationships belonging to the specified digital twin and the http response.
-     * TODO: Impl here returns an Object and not a String.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedFlux<Object> listRelationships(String digitalTwinId, String relationshipName) {
-        return new PagedFlux<>(
-            () -> protocolLayer.getDigitalTwins().listRelationshipsSinglePageAsync(digitalTwinId, relationshipName),
-            nextLink -> protocolLayer.getDigitalTwins().listRelationshipsNextSinglePageAsync(nextLink));
+    public PagedFlux<String> listRelationships(String digitalTwinId, String relationshipName) {
```

Comment:
The final relationship APIs have not been implemented yet. They will be implemented in the next PR.
While the final pageable API will look similar to this, this is more of a test to see how the PL works.